app/controllers/katello/api/v2/repository_sets_controller.rb

```diff
@@ -100,6 +100,7 @@ def enable
     api :PUT, "/repository_sets/:id/disable", N_("Disable a repository from the set")
     api :PUT, "/products/:product_id/repository_sets/:id/disable", N_("Disable a repository from the set")
     param :id, :number, :required => true, :desc => N_("ID of the repository set to disable")
+    param :repo_id, :number, :required => false, :desc => N_("ID of the repository within the set to disable")
```

Comment:
is this used?

Reply:
is the idea that the user could just pass in the repo_id in case other stuff isn't working?

Reply:
could we change this to repository_id ?

Reply:
i do like this approach though, it means its always able to be disabled from the UI!!!